code/modules/spells/spell_types/conjure.dm

```diff
@@ -82,19 +82,35 @@
 	cooldown_min = 10
 	var/delete_old = TRUE //TRUE to delete the last summoned object if it's still there, FALSE for infinite item stream weeeee
 
-/obj/effect/proc_holder/spell/targeted/conjure_item/cast(list/targets, mob/user = usr)
-	if (delete_old && item && !QDELETED(item))
-		QDEL_NULL(item)
+/obj/effect/proc_holder/spell/aoe_turf/horde
+	name = "Horde"
+	desc = "Bring all your existing bloodmen to you at the cost of 18% blood (15 brain damage for those without blood)."
+	action_icon = 'icons/mob/actions/actions_cult.dmi'
+	action_icon_state = "horde"
+	var/list/summon_type = list("/mob/living/simple_animal/hostile/asteroid/hivelord/legion/bloodman") //determines what exactly will be summoned
+	//should be text, like list("/mob/living/simple_animal/bot/ed209")
+	clothes_req = FALSE
+	var/summon_lifespan = 0 // 0=permanent, any other time in deciseconds
+	var/summon_ignore_density = FALSE //if set to TRUE, adds dense tiles to possible spawn places
+	var/summon_ignore_prev_spawn_points = TRUE //if set to TRUE, each new object is summoned on a new spawn point
+
+	var/list/newVars = list() //vars of the summoned objects will be replaced with those where they meet
+	//should have format of list("emagged" = 1,"name" = "Wizard's Justicebot"), for example
+
+/obj/effect/proc_holder/spell/aoe_turf/horde/cast(list/targets,mob/living/carbon/user = usr)
+	if(GLOB.bloodmen_list.len < 1)
+		to_chat(user, span_notice("You don't have any minions to summon!"))
+		return
+	if(NOBLOOD in user.dna.species.species_traits)
+		to_chat(usr, span_notice("<span class ='userdanger'>You can almost feel your brain writhing as you call your bloodmen to you.</span>"))
+		user.adjustOrganLoss(ORGAN_SLOT_BRAIN, 15)
 	else
-		for(var/mob/living/carbon/C in targets)
-			if(C.dropItemToGround(C.get_active_held_item()))
-				C.put_in_hands(make_item(), TRUE)
-
-/obj/effect/proc_holder/spell/targeted/conjure_item/Destroy()
-	if(item)
-		qdel(item)
-	return ..()
-
-/obj/effect/proc_holder/spell/targeted/conjure_item/proc/make_item()
-	item = new item_type
-	return item
+		to_chat(user, span_notice("<span class ='userdanger'>You feel yourself becoming pale as you call your minions to you.</span>"))
+		user.blood_volume -= 100 //most expensive because you're calling up to 3 bloodmen, like 18% blood cost
```

Comment:
This could be put inside the for loop to be a cost per bloodman

Reply:
makes more sense than charging 18% blood for one dude